src/Entities/Transformers/Family.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Choredo\Entities\Transformers;
+
+use Assert\Assertion;
+use Choredo\Entities;
+use League\Fractal\TransformerAbstract;
+
+class Family extends TransformerAbstract
+{
+    const DAYS_OF_WEEK = ['sunday', 'monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday'];
```

Comment:
This already exists here: https://github.com/GreyBee/choredo-api/blob/master/src/Entities/Schedule.php#L7

I'd rather use that constant than repeat it. What do you think?